src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsIncontentBoxadAE3MobileWiki.java

```diff
@@ -0,0 +1,91 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Test(groups = "AdsIncontentBoxadAE3Mobile")
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class TestAdsIncontentBoxadAE3MobileWiki extends NewTestTemplate {
+
+  private static final Page page = new Page("project43", "SyntheticTests/LongPage");
+
+  private static final String AD_INFO_PATTERN = "https?://.*wikia-services\\.com.*kv_pos=INCONTENT_BOXAD.*kv_rv=4.*";
```

Comment:
It would be great to assert tracking of last element (viewport-conflict, once user scroll down to global footer)